Documented new env vars for Contao 4.10 (#488)
contao/docs: docs/dev/reference/config.md

However, protecting your users from CSRF attacks but let them submit the form via unsecured `http` connections is
not really a valid use case. 
{{% /notice %}}

### `COOKIE_REMOVE_FROM_DENY_LIST`

{{< version "4.10" >}}

> In case you don't want to manage the whole `COOKIE_ALLOW_LIST` because you're unsure what your application needs but

Reviewer: ```suggestion
In case you don't want to manage the whole `COOKIE_ALLOW_LIST` because you are unsure what your application needs but
```

you want to disable one of the existing entries on the deny list that's managed by Contao, you can specify this using:

```

> COOKIE_REMOVE_FROM_DENY_LIST=__utm.+

Reviewer: Is this env var just a comma separated list? If yes, may be show that in the example :)

Author: Done

```

### `QUERY_PARAMS_ALLOW_LIST`

{{< version "4.10" >}}

For the very same reason we strip irrelevant cookies, we also strip irrelevant query parameters. E.g. you might be
familiar with the typical `?utm_*>=<randomtoken>` query parameters that are added to links of your website. Because they
change the URL every single time, they also generate new cache entries every single time, eventually maybe even flooding
your cache.

As with the irrelevant cookies, Contao also manages a list of irrelevant query parameters which again, you may completely
override by providing a list of allowed query parameters if you know all the query parameters your application ever

> needs. This is highly unlikely which is why there's also `QUERY_PARAMS_REMOVE_FROM_DENY_LIST`.

Reviewer: ```suggestion
needs. This is highly unlikely which is why there is also `QUERY_PARAMS_REMOVE_FROM_DENY_LIST`.
```

### `QUERY_PARAMS_REMOVE_FROM_DENY_LIST`

{{< version "4.10" >}}

As with `COOKIE_REMOVE_FROM_DENY_LIST`, you can use `QUERY_PARAMS_REMOVE_FROM_DENY_LIST` to remove an entry from the
default deny list shipped with Contao. If you e.g. need the Facebook click identifier (`fbclid`) in your server side
code, you may update your list like so:

```
QUERY_PARAMS_REMOVE_FROM_DENY_LIST=fbclid
```

{{% notice warning %}}
If you do so, make sure to disable caching by e.g. setting `Cache-Control: no-store` on this response if `fbclid` is

> present as otherwise you're back to having thousands of cache entries in your cache proxy.

Reviewer: ```suggestion
present as otherwise you are back to having thousands of cache entries in your cache proxy.
```
